github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1293,3 +1293,60 @@ periodics:
       - name: pgp-bot-key
         secret:
           secretName: pgp-bot-key
+- name: periodic-kubevirt-e2e-k8s-1.21-sig-performance
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "40 7,15,23 * * *"
+  decorate: true
+  cluster: prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-dind-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: main
+    work_dir: true
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20210311-09ebaa2
+        env:
+          - name: KUBEVIRT_PROVIDER
+            value: "k8s-1.21"
+          - name: KUBEVIRT_STORAGE
+            value: "hpp"
+          - name: KUBEVIRT_MEMORY_SIZE
+            value: "9G"
+          - name: KUBEVIRT_NUM_NODES
+            value: "4"
+          - name: KUBEVIRT_DEPLOY_PROMETHEUS
+            value: "true"
+        command:
+          - "/usr/local/bin/runner.sh"
+          - "/bin/sh"
+          - "-c"
+          - |
+            # Create k8s cluster
+            make cluster-up
+
+            # Push and deploy kubevirt
+            make cluster-sync
+
+            FUNC_TEST_ARGS="--noColor --seed=42" make perftest            
+        # docker-in-docker needs privileged mode
+        securityContext:
+          privileged: true
+        resources:
+          requests:
+            memory: "36Gi"
```

Comment:
Consider adding a  few more GB to the request to avoid OOM kills, since the VMs already use 4*9G. Maybe add 8 more?

Reply:
Right, 8G more sounds reasonable!

Reply:
I will also include 1 more CPU. 4*6+1

Reply:
Hm, we normally don't add cpu requests at the moment. I would at least suggest to request only a part of the CPUs.

Reply:
hum, can it be 4*6?
I am wondering it could become a flaky test when there are not enough CPU, so that VMIs will not be scheduled.

Reply:
All other jobs don't reqeust any CPU resources at all (using the default of 100m implicitly). We control the job density right now through the memory requests (since there are also other aspects like disk io which can't be managed on that level). This can probably still be improved, but I think it is best if you just leave that out right now. You can still request the normal amount of vcpus in for the qemu nodes (just leaving the default like you do right now).